WordPress/Classes/Services/ReaderTopicService+FollowedInterests.swift

```diff
@@ -83,10 +83,9 @@ extension ReaderTopicService: ReaderFollowedInterestsService {
     }
 
     private func apiRequest() -> WordPressComRestApi {
-        let defaultAccount = self.coreDataStack.performQuery { context in
-            try? WPAccount.lookupDefaultWordPressComAccount(in: context)
+        let token = self.coreDataStack.performQuery { context in
+            try? WPAccount.lookupDefaultWordPressComAccount(in: context)?.authToken
         }
-        let token: String? = defaultAccount?.authToken
```

Comment:
Tell me if I got this right: The problem this change addresses is that returning an `NSManagedObject` from `performQuery` might lead to thread-unsafe accesses (for lack of a better word).

Reply:
Spot on!